src/components/Combobox/components/StickyCategoryHeader/StickyCategoryHeader.module.scss

```diff
@@ -0,0 +1,9 @@
+.stickyCategoryHeader {
+  position: absolute;
+  z-index: 10;
+  width:  calc(100% - 12px);
```

Comment:
what is this 12px. Can it be dynamic?

Reply:
I did not found a way to make it dynamic.
the 12 px its the outside margin size and the fixed scroll size. if the width of the sticky category was 100% it would hide the scrollbar (because the element is outside of the virtualized list)
I can add note which explain this